.babelrc

```diff
@@ -1,4 +1,9 @@
 {
-  "presets": ["es2015"],
-  "plugins": ["transform-object-assign"]
+  "presets": [
+    ["@babel/preset-env", {
+      "targets": {
+        "browsers": ["last 2 versions", "IE 10"]
```

Comment:
this change was already in the branch. not sure if you prefer to keep it, but fwiw: https://jamie.build/last-2-versions